src/editor/assets/asset-panel.ts

```diff
@@ -262,6 +287,260 @@ class AssetGridViewItem extends GridViewItem {
  * @property {boolean} writePermissions If false then only a read-only view will be shown
  */
 class AssetPanel extends Panel {
+    // Static properties
+    static VIEW_LARGE_GRID = 'lgrid';
+
+    static VIEW_SMALL_GRID = 'sgrid';
+
+    static VIEW_DETAILS = 'details';
+
+    static LEGACY_SCRIPTS_ID = LEGACY_SCRIPTS_ID;
+
+    /**
+     * @type {Container}
+     */
+    _containerControls;
```

Comment:
Why is this not just:

```ts
    _containerControls: Container;
```

Reply:
I was hoping the JSDoc types improved the Intellisense issues you found. It's just a search replace to change the jsdoc tags to TS types

Reply:
Yeah, we don't have any plans to work on that ourselves at the moment (unless someone's working on a particular file for some other reason and makes some TS updates while they're there). If any community members feel like doing TS-related updates, be my guest.